tools/replay-archive/src/cli.rs

```diff
@@ -81,17 +81,8 @@ impl ReplayController {
         let chain_store = ChainStore::new(store.clone(), genesis_height, false);
 
         let head_height = chain_store.head().context("Failed to get head of the chain")?.height;
-        let tail_height = chain_store.tail().context("Failed to get tail of the chain")?;
-        assert_eq!(
```

Comment:
removed this assert since it is not correct (tail is not necessarily same as genesis height)